components/bms_worker/src/CAN/CANIO-tx.c

```diff
@@ -66,26 +69,32 @@ static const packTable_S* packNextMessage(const packTable_S* packTable,
                                           CAN_data_T*        message,
                                           uint8_t*           nextCounter);
 
-static bool MSG_pack_BMS_100Hz(CAN_data_T* message, const uint8_t counter);
-static bool MSG_pack_BMS_100Hz1(CAN_data_T* message, const uint8_t counter);
-static bool MSG_pack_BMS_100Hz2(CAN_data_T* message, const uint8_t counter);
-static bool MSG_pack_BMS_100Hz3(CAN_data_T* message, const uint8_t counter);
+static bool MSG_pack_BMS_100Hz_Critical(CAN_data_T* message, const uint8_t counter);
```

Comment:
the format for these function names should be as follows:
`pack_BUS_NODE_messageName_period`
i.e. if the message name is BMS_cellData on the VEH bus at 10ms, the name should be`:
`pack_VEH_BMS_cellData_10ms`